### DISCUSSION TITLE
Zigbee: IKEA Tradfri E1766 Manufacturer contains invalid characters

### DISCUSSION THREAD

**Discussion by sisamiwe** [arendst/Tasmota#11886] [Q&A] [answered]
Hi,

I have connected a IKEA Tradfri E1766 Remote zu the SONOFF Zigbee Bridge running with Tasmota 9.4.0 Zigbee.

The reported Manufacturer contains invalid letters. It is shown as "KE" and can therefore not processed.
`tele/D1B8/ZbReceived = {"0xD1B8":{"Device":"0xD1B8","ModelId":"TRADFRI open/close remote","Manufacturer":"KE","Endpoint":1,"LinkQuality":68}}`

Where this can be corrected?

Thank you

**Comment by s-hadinger** [comment, marked as answer]
One thing you can do is `ZbStatus2 0xD1B8`

Then copy the JSON of the device, fix the Manufacturer and do `ZbRestore <json>`.

For example with my bulb:
`ZbRestore {"Device":"0xCFD5","Name":"IKEA_Bulb","IEEEAddr":"0x90FD9FFFFE03B051","ModelId":"TRADFRI bulb E27 WS opal 980lm","Manufacturer":"IKEA of Sweden","Endpoints":[1],"Config":["L01.2","O01"]}`

**Comment by sisamiwe** [comment]
@s-hadinger 

Thanks for your response.
It worked!
